### ISSUE TITLE
[Feature request]: Jackd support for glassplayergui

### ISSUE THREAD

**Issue by alexolivan** [RadioFreeAsia/GlassPlayer#1]
Hi!

Glassplayergui seems to me an awesome idea In a world very so video-focused... a simple GUI to keep an eye on an incoming audio streaming into the DAW... awesome... but I miss that, while it is fairly easy to use the glassplayer CLI binary to connect to jackd (and therefore, add the audio input into the DAW) I miss not being able to do so through glassplayergui.

It would be nice if we could pass to gplassplayer through glassplayergui a few arguments (I'm just learning C++ and barely no Qt knowledge , but I think glassplayergui spawns glassplayer but I see no arguments beyond url).... I guess the hard part would be to get metering from jack.

Regards.

**Comment by alexolivan** [comment]
By adding args.push_back("--audio-device=JACK");  at processStart turns out that this works out of the box...so very much possible.

Now I'm going to see If I manage to make it to intercept the jack type, jack server name and jack client name arguments and pass them to processStart as an aditional argument... very fun! :-D

Thank you very much for such nice pieces of code!!!

EDIT: 
Managed to do what I want... My coding skill are very noob level, but it works...
Now I can, optionally, pass any combination of the 3 jackd related arguments like this (once --audio-device=JACK is added):
```
glassplayergui --audio-device=JACK --jack-server-name=default --jack-client-name=myradio http://dnas.myradio.com:8000
```
I have also appended the optional client name to the title bar (like in GlassCoder) 
If no arguments are provided, it behaves as before

**Comment by ElvishArtisan** [comment]
Implemented in #3.
